oneflow/python/framework/tensor.py

```diff
@@ -13,12 +13,269 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 """
-import oneflow_api
-import oneflow.python.framework.dtype as dtype_util
 from oneflow.python.oneflow_export import oneflow_export
 import oneflow.python.framework.device as oneflow_device
+import oneflow_api
+
+
+@oneflow_export("tensor")
+class Tensor:
+    def __init__(
+        self,
+        shape,
```

Comment:
这个构造方法和 pytorch 没有对齐，

```
>>> import torch
>>> torch.Tensor(1, 2)
tensor([[8.8368e-14, 4.5673e-41]])
>>> torch.Tensor([1, 2])
tensor([1., 2.])
>>> torch.Tensor((1, 2))
tensor([1., 2.])
```

用一个错误的参数列表触发的它的错误提示：

> TypeError: new() received an invalid combination of arguments - got (int, int, dtype=torch.dtype), but expected one of:
>  * (*, torch.device device)
>  * (torch.Storage storage)
>  * (Tensor other)
>  * (tuple of ints size, *, torch.device device)
>       didn't match because some of the keywords were incorrect: dtype
>  * (object data, *, torch.device device)
>       didn't match because some of the keywords were incorrect: dtype